storyboarding/scripting/compound_commands.rst

```diff
@@ -12,6 +12,405 @@ Loop
+   :header: "Condition Type", "Name of Sampleset", "Additional", "Number of Sampleset", "Triggername"
+   :widths: auto
+
+   "Hitsound", "Soft", "Whistle", "1", "HitsoundSoftWhistle1"
+   "Hitsound", "Drum", "", "", "HitsoundDrum"
+   "Hitsound", "", "Clap", "", "HitsoundClap"
+   "Hitsound", "", "", "4", "Hitsound4"
+   "Hitsound", "Normal", "", "2", "HitsoundNormal2"
+   "Hitsound", "", "Finish", "0", "HitsoundFinish0"
+
+The columns are very easy to understand when you are familiar with hitsounding. Aside from the prefix "Hitsound" you can individually add or leave out a specification of your hitsound to get the exact hitsound or combination of hitsounds you want.
+
+If you don't know what any of these mean, consider reading `a guide on hitsounding <https://osu.ppy.sh/s/224758>`_.
+
+ .. warning:: While the naming of the triggers suggests that it checks which hitsound is played, this is actually not the case!
+    Changing the sampleset of a note via the per-note quickaccess on the top left of the editor(Shift+Q/W/E/R) or via Sample Import will be ignored for the sake of evaluating triggers.
+    You **always** have to change your samplesets via inherited timing sections if you want your Hitsound triggers to work correctly.
+
+.. note:: You might have noticed that there is no option to catch a hitnormal alone. This is very annoying when you want to catch hitnormals along with other notes that actually have finishers.
+  It can be worked around by changing the sampleset for these notes so you can use ``Hitsound5`` or something like that but it requires a lot of manual effort nonetheless.
+
+  If you wonder why this is the case, the answer is that hitnormals work for everyone differently. Currently there is the option to configure the option ``LayeredHitSounds`` in your ``Skin.ini`` to 0 causing hitnormals no longer be played on notes that have additionals.
+  As this is a setting determined by the user in most cases, there is no way to get a consistent experience for all players without the already mentioned workaround anyway.
+
+Example: Hitsounds
+..................
+Aoba wants to play the Taiko drum! She is very inexperienced but maybe you can help her out?
+
+.. image:: img/compound_commands/aobaTaiko.png
+  :scale: 20%
+  :alt: Full of optimism: Aoba
+  :align: left
+
+.. image:: img/compound_commands/drum.png
+  :scale: 30%
+  :alt:  ...and the Taiko drum!
+  :align: right
+
+If you aren't familiar with Taiko mapping, no problem, we got that covered in a few words.
+
+There are 2 types of notes in Taiko, Don (red) and Kat (blue). If it has a whistle and/or clap hitsound attached it is a Kat, otherwise it is a Don note.
+
+
+We can make Aoba hit the drum by catching the respective trigger conditions:
+
+These would be ``HitsoundWhistle`` and ``HitsoundClap`` for Kat.
+
+For Don we have to create a workaround and give each note a separate Hitsoundset because we can't react to the hitnormal without reacting to the additionals as well.
```

Comment:
While I agree that this should usually be avoided, the frustration and tears on my own journey  of getting triggered led me to believe that it is most important to know what does `not` work with triggers and how you can circumvent it rather than having some theoretical knowledge that runs you into a wall after you finished your conception.

Reply:
I think in a future version of the article (maybe when it's split into its own page), we can add a basic example first, then have the next two examples be two scenarios demonstrating triggers' usefulness and shortcomings.